sanic_prometheus/__init__.py

```diff
@@ -61,7 +61,8 @@ def monitor(app, endpoint_type='url:1',
             get_endpoint_fn=None,
             latency_buckets=None,
             mmc_period_sec=30,
-            multiprocess_mode='all'):
+            multiprocess_mode='all',
+            memcollect_enabled=True):
```

Comment:
How about we reuse `mmc_period_sec` for enabling/disabling memory metrics collector? If mmc_period_sec is set to `None` we won't launch the task.

And the docstring could be something like:
```
     :param mmc_period_sec: set a period (in seconds) of how frequently memory
                            usage related metrics are collected.  
                            Setting it to None will disable memory metrics collection. 
```

Reply:
Done